crates/nu-cli/src/examples.rs

```diff
@@ -20,6 +20,7 @@ pub fn test(cmd: impl WholeStreamCommand + 'static) {
         whole_stream_command(Each {}),
         whole_stream_command(cmd),
         whole_stream_command(StrCollect),
+        whole_stream_command(FromJSON),
```

Comment:
We try to limit the commands here. Can you please remove this and use table literals in the examples instead? Thank you!